readme.md

```diff
@@ -0,0 +1,10 @@
+# Watson Discovery News Crypto Sentiment Analysis
+
+In this example, we will use Watson Discovery News in order to query articles written about Bitcoin and visualize the sentiment of the articles. This should act as a baseline for integrating Watson Discovery into your own application. The use cases are endless, with Watson Discovery, and especially the "news" collection, you can harness the power of advanced queries in order to gain many insights.
+
+![](images/flow.png)
+
+1. User signs up for IBM Cloud
```

Comment:
we typically document the process flow / overall architecture for a code pattern. not so much the user flow.

in this case, i was expecting it to be more like... 1. user runs a notebook on watson studio, 2. the notebook calls out to watson discovery, 3. the notebook renders data retrieved from disco news service and does blah blah blah, 4. the user sees the visualizations and can do x